cli.js

```diff
@@ -115,3 +123,14 @@ napa._loadFromPkg = function (property, defaults) {
 napa.getref = function (url) {
   return url.replace(/^[^#]*#?/, '')
 }
+
+napa._findFlags = function (args) {
```

Comment:
Could we use a library like [minimist](https://www.npmjs.com/package/minimist) instead?